style.css

```diff
@@ -0,0 +1,59 @@
+body {
+  background-color: azure;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  text-align: center;
+}
+
+.form-main {
+  text-align: left;
+  margin-bottom: 25px;
+}
+
+form {
+  background-color: #fff;
+  max-width: 500px;
+  margin: 50px auto;
+  padding: 30px 20px;
+  box-shadow: 2px 5px 10px rgba(0, 0, 0, 0.5);
+}
+
+.form-main label {
+  display: inline-block;
+  text-align: left;
+  vertical-align: top;
+  width: 48%;
+}
+
+.form-main input,
+.form-main select,
+.form-main textarea {
+  border: 1px solid #777;
+  border-radius: 2px;
+  font-family: inherit;
+  padding: 10px;
+  display: block;
+  width: 95%;
+}
+
+.form-main input[type='radio'],
+.form-main input[type='checkbox'] {
+  display: inline-block;
+  width: auto;
+}
+
+button {
+  background-color: moccasin;
+  border: 1px solid #777;
+  border-radius: 5px;
+  font-family: inherit;
+  font-size: 21px;
+  display: block;
+  width: 100%;
+  margin-top: 50px;
+  margin-bottom: 20px;
+}
+
+button:hover {
+  background-color: azure;
+  cursor: pointer;
+}
```

Comment:
1. Please instead of using (border: 1px solid #777;) twice (that is on line 30 and line 46), I suggest you create a class and add it to all elements that need to be.

2. You should do the same for the background-color: azure; which has been used on line 2 and line 57